user-impl/src/main/java/com/example/auction/user/impl/UserRepository.java

```diff
@@ -0,0 +1,180 @@
+                    .selectAll(
+                            "SELECT * FROM User " +
+                                    "ORDER BY  UserId DESC " +
+                                    // ORDER BY status is required due to https://issues.apache.org/jira/browse/CASSANDRA-10271
+                                    "LIMIT ?",
+                            limit
+                    )
+                    .thenApply(List::stream)
+                    .thenApply(rows -> rows.skip(offset))
+                    .thenApply(rows -> rows.map(UserRepository::convertUserSummary))
+                    .thenApply(UserSummaries -> UserSummaries.collect(Collectors.toList()))
+                    .thenApply(TreePVector::from);
+        }
+
+
+    private static User convertUserSummary(Row user) {
+        return new User(
+                user.getUUID("UserId"),
+                user.getString("Name"),
+                user.getString("email")
+        );
+    }
+        private static class PUserEventProcessor extends ReadSideProcessor<PUserEvent> {
+
+            private final CassandraSession session;
+            private final CassandraReadSide readSide;
+
+            private PreparedStatement insertUserStatement;
+
+            @Inject
+            public PUserEventProcessor(CassandraSession session, CassandraReadSide readSide) {
+                this.session = session;
+                this.readSide = readSide;
+            }
+
+            @Override
+            public ReadSideHandler<PUserEvent> buildHandler() {
+                return readSide.<PUserEvent>builder("pUserEventOffset")
+                        .setGlobalPrepare(this::createTables)
+                        .setPrepare(tag -> prepareStatements())
+                        .setEventHandler(PUserEvent.PUserCreated.class,
+                                e -> insertUser(e.getUser()))
+
+                        .build();
+            }
+
+            @Override
+            public PSequence<AggregateEventTag<PUserEvent>> aggregateTags() {
+                return PUserEvent.TAG.allTags();
+            }
+
+            private CompletionStage<Done> createTables() {
+                return doAll(
+                        session.executeCreateTable(
+                                "CREATE TABLE IF NOT EXISTS UserInfo (" +
+                                        "UserId UUID PRIMARY KEY, " +
+                                        "Name text, "+
+                                        "email text, " +
+                                        "PasswordHash text "+
+                                        ")"
+                        )
+
+                );
+            }
+
+            private CompletionStage<Done> prepareStatements() {
+                return
+                        prepareInsertUserStatement();
+
+
+            }
+
+
+
+            private CompletionStage<Done> prepareInsertUserStatement() {
+
+                return session.
+                        prepare("INSERT INTO User(" +
```

Comment:
The name of the table here ("User") does not match the table created above ("UserInfo").